entries/u/united.com.json

```diff
@@ -2,12 +2,20 @@
   "United": {
     "domain": "united.com",
     "url": "https://www.united.com/",
-    "contact": {
-      "facebook": "United",
-      "twitter": "united"
-    },
-    "categories": [
+    "tfa": [
+      "email",
+      "sms",
+      "call",
+      "totp",
```

Comment:
```suggestion
```

The United app's use as an authenticator does not constitute TOTP because it is [a proprietary software method](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#other-properties). TOTP means that it can be used in any authenticator app, like Google Authenticator or Authy.